imports/plugins/core/dom/client/dom.js

```diff
@@ -0,0 +1,50 @@
+// import { Tracker } from "meteor/tracker";
```

Comment:
Remove this comment

Reply:
comment removed.

Reply:
and logic in metadata.js has been refactor to address your comment above.